
Increase the default TTL of SpoofAction records until the Lua binding is fixed
to properly accept the options parameter with the single-address argument form.

diff -rup dnsdist-1.5.1/dnsdist-lua.hh dnsdist-1.5.1-ttl/dnsdist-lua.hh
--- dnsdist-1.5.1/dnsdist-lua.hh	2020-09-30 13:09:33.000000000 +0200
+++ dnsdist-1.5.1-ttl/dnsdist-lua.hh	2020-11-22 12:58:24.227139530 +0100
@@ -28,7 +28,7 @@ struct ResponseConfig
   boost::optional<bool> setAA{boost::none};
   boost::optional<bool> setAD{boost::none};
   boost::optional<bool> setRA{boost::none};
-  uint32_t ttl{60};
+  uint32_t ttl{3600};
 };
 void setResponseHeadersFromConfig(dnsheader& dh, const ResponseConfig& config);
 
